pyxrf/model/xrf_energy.py

```diff
@@ -0,0 +1,21 @@
+
+from .load_data_from_db import make_hdf
+from .command_tools import pyxrf_batch
+
+import logging
+logger = logging.getLogger()
+
+
+def xrf_energy_batch(*args, **kwargs):
+    try:
+        xrf_energy_batch_api(*args, **kwargs)
+    except BaseException as ex:
+        msg = f"Processing is incomplete! Exception was raised during execution:\n   {ex}"
```

Comment:
I am not sure. My idea for ``build_xanes_maps`` was to avoid printing the stack trace. Rerunning with ``build_xanes_map_api`` will print the stack trace. If community decides that stack trace should always be printed, then ``build_xanes_map_api`` should be renamed to ``build_xanes_maps``.